adapter/config/parser.go

```diff
@@ -56,8 +54,6 @@ const (
 	relativeConfigPath = "/conf/config.toml"
 	// RelativeLogConfigPath is the relative file path where the log configuration file is.
```

Comment:
We dont need relative config path now. So can you remove that?

Reply:
removed the relativeLogConfigPath